LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019 Cartesi Pte. Ltd.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Need to fix this

Reply:
Fix what? This is an appendix to the license. It is not something we should change.

Reply:
Sorry my bad. Thought it was part of source code file, not the License appendix